src/open_mower/launch/open_mower.launch

```diff
@@ -9,7 +9,7 @@
 
     <node pkg="mower_map" type="mower_map_service" name="map_service" output="screen"/>
     <node pkg="mower_logic" type="mower_logic" name="mower_logic" output="screen">
-        <param name="automatic_start" value="$(env OM_AUTOMATIC_START)"/>
+        <param name="automatic_mode" value="$(env OM_AUTOMATIC_MODE)"/>
```

Comment:
please replace with `optenv OM_AUTOMATIC_MODE 0` in order to not break existing mowers (using env with an undefined environment variable will fail to start the software).
```suggestion
        <param name="automatic_mode" value="$(optenv OM_AUTOMATIC_MODE 0)"/>
```